policyengine_us/variables/gov/states/co/tax/income/credits/care_worker/co_care_worker_credit.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class co_care_worker_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Care Worker Tax Credit"
+    defined_for = "co_care_worker_credit_eligible"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/colorado/title-39/specific-taxes/income-tax/article-22/part-5/section-39-22-566/"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.co.tax.income.credits.care_worker
+        joint = tax_unit("tax_unit_is_joint", period)
+        eligible_care_worker_count = add(
+            tax_unit, period, ["co_care_worker_credit_eligible_care_worker"]
```

Comment:
i think this implies only head and spouse:

>In the case of a child care worker or qualified direct care worker who **files a joint return with** another resident individual who is also a child care worker or a qualified direct care worker

Reply:
Agreed, adding the head and spouse condition